gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -64,10 +64,10 @@ def get_shared_images
   end
 
   def get_public_images
+    filters = @options.public_images_filters
```

Comment:
sure, will do :) it is just an early PR to talk about how to do this, will definitely add tests